DashAI/back/main.py

```diff
@@ -1,42 +1,56 @@
-import json
+import logging
+import sys
 
 import uvicorn
-from fastapi import Body, FastAPI
-from fastapi.middleware.cors import CORSMiddleware
+from fastapi import FastAPI, Request
+from fastapi.responses import RedirectResponse
+from fastapi.staticfiles import StaticFiles
+from fastapi.templating import Jinja2Templates
+from sqlalchemy.exc import DBAPIError, SQLAlchemyError
+from sqlalchemy.sql import text
 
-from DashAI.back.config_object import ConfigObject
 from DashAI.back.database import db
-from DashAI.back.models.classes.getters import filter_by_parent
-from DashAI.back.models.enums.squema_types import SquemaTypes
 from DashAI.back.routers import datasets, experiments
-from DashAI.back.routers.session_class import session_info
 
 # TODO These imports should be removed because they are unused, but currently needed.
 from DashAI.back.tasks.tabular_classification_task import TabularClassificationTask
 from DashAI.back.tasks.task import Task
 from DashAI.back.tasks.text_classification_task import TextClassificationTask
 from DashAI.back.tasks.translation_task import TranslationTask
 
+logging.basicConfig(level=logging.DEBUG)
+log = logging.getLogger(__name__)
+
 app = FastAPI(title="DashAI")
 
-origins = [
-    "http://localhost:3000",
-]
 
-app.add_middleware(
-    CORSMiddleware,
-    allow_origins=origins,
-    allow_credentials=True,
-    allow_methods=["*"],
-    allow_headers=["*"],
-)
+@app.get("/")
+async def redirect():
+    return RedirectResponse(url="/app")
+
+
```

Comment:
Perhaps for the backlog, any route that is not registered in any router should be returned by the application. 
So if for example I enter localhost /asddsd it does not deliver a bad request directly to the server but returns the app.

Reply:
Now localhost/asddsd and localhost/ will redirect to /app/
BUT localhost/app/asddsd should be handled by React routing and not by the backend endpoint.